apps/XMRig/install-32

```diff
@@ -0,0 +1,200 @@
+#!/bin/bash
+version=6.16.2
+
+enable_64bit() {
+sudo cat /boot/config.txt &>/dev/null
```

Comment:
Wouldn't it be simpler to check that /boot/config.txt exists? Not sure why `cat` is being used here.
```bash
if [ -f /boot/config.txt ]
```

Reply:
I'll change that.